pkg/system/phase2_creating.go

```diff
@@ -258,6 +258,21 @@ func (r *Reconciler) SetDesiredCoreApp() error {
 					if r.OAuthEndpoints != nil {
 						c.Env[j].Value = r.OAuthEndpoints.TokenEndpoint
 					}
+
```

Comment:
if the proxies envs are not set on the operator pod then core\endpoints are deployed with empty vars (just names and no value). 
does that work?

Reply:
yes it does work

Reply:
I think there is a subtle difference between empty vars and non-existing vars. I am not an expert on that, but I think it's better to forward the correct value. you can use `LookupEnv` to distinguish between the cases.

from the documentation of Getenv:
```
// Getenv retrieves the value of the environment variable named by the key.
// It returns the value, which will be empty if the variable is not present.
// To distinguish between an empty value and an unset value, use LookupEnv.
```